commands/build.go

```diff
@@ -41,6 +42,7 @@ func init() {
 	buildCmd.Flags().StringVar(&language, "lang", "", "Programming language template")
 
 	// Setup flags that are used only by this command (variables defined above)
+	buildCmd.Flags().StringVar(&ssh, "ssh", "", `(Experimental) The value to be passed to the '--ssh' flag of 'docker build'.`)
```

Comment:
If we add this flag, I think it would be good to extend this doc string to mention that BUILDKIT is required, maybe even directly mentioning the `DOCKER_BUILDKIT=1` var

Reply:
Oh does this *only* work with buildkit?

Reply:
+1 to Lucas' comment.

If so, it might be best to add the `DOCKER_BUILDKIT` env to the build command.

Reply:
not only does it require buildkit, but you have use the experimental syntax in the Dockerfile `# syntax = docker/dockerfile:1.0-experimental`  and update the relevant RUN commands like this https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/experimental.md#run---mounttypessh

Reply:
It doesn't _require_ Buildkit...

```
$ cat Dockerfile
FROM golang:1.13.4 AS builder
COPY . .
RUN go build -o /app main.go

FROM gcr.io/distroless/base:nonroot
COPY --from=builder /app /app
CMD /app

$ docker build --ssh default -t foo:bar .
(...)
Successfully tagged foo:bar
```

Reply:
It won't fail without buildkit, but you must use the experimental mode/syntax to utilize the ssh feature, right?

Reply:
Yeah, right—hence the default being `false` 🙂

Reply:
To be clear, I don't want to block accepting this PR, only improve the doc string to provide a stronger hint to people that experimental here is refers directly to an experimental, docker/buildkit feature.

This isn't _our_ experimental feature, it is Docker's experimental feature that we are exposing.